diff --git a/TripitakaPlatform/urls.py b/TripitakaPlatform/urls.py
index 43df438..b937f1a 100644
--- a/TripitakaPlatform/urls.py
+++ b/TripitakaPlatform/urls.py
@@ -24,7 +24,6 @@ from tasks.views.mark import *
 from tasks.views import lqtripitaka
 from tasks.views import tripitaka
 from tools.views import *
-from tdata.views import email_vericode
 import xadmin
 import tdata
  
@@ -34,8 +33,6 @@ urlpatterns = [
     path('manage/', xadmin.site.urls),
     url(r'^auth/', include("jwt_auth.urls", namespace="api-auth")),
     url(r'^api/', include('api.urls')),
-    url(r'^auth/api-vericode/', email_vericode),
-    url(r'^activate/(?P<token>[0-9a-zA-Z@.*$]+)/$', tdata.views.active_user, name='index'),
     path('correct/<int:task_id>/', do_correct_task, name='do_correct_task'),
     path('verify_correct/<int:task_id>/', do_correct_task, name='do_correct_verify_task'),
     path('correct_difficult/<int:task_id>/', do_correct_task, name='do_correct_task'),
